config/optional/rest.resource.entity.paragraphs_row.yml

```diff
@@ -0,0 +1,17 @@
+langcode: en
```

Comment:
Not sure this should be optional after reading through the rest of the code. Arguments for leaving it here?

Reply:
meh. :shrug:

Reply:
You have code that is reliant on this config so it isn't really optional in that case. I would suspect you do want to have conflicts show up with other modules if they implement the same config. It would be more clear to whoever wins that way.

Reply:
:tomato: :tomato: 
:potato: :potato: